Tests/Tests/MainContainerSpec.swift

```diff
@@ -22,24 +22,25 @@ class MainContainerSpec: QuickSpec {
             it("will have all dependencies resolved") {
                 let instances: [Any?] = [
                     dependencyManager.resolve(type: CommonUtility.self),
```

Comment:
What's the relevance of the dep mgr changes?

Reply:
Here, it's just a reorder to mach the order in MainContainer.swift.
The main fix is in `getValidConfigURL()` method. Using `URL(string: "config.com")` in Reachability fails because `host` cannot be determined for that kind of url. This makes Reachability init to return nil which fails the test.
I don't know why it worked before. Maybe because of changes iOS SDK version